Replace all http instances with https

MylesJarvis opened the pull request:
### Trello card

### Context

### Changes proposed in this pull request

### Guidance to review

MylesJarvis committed:
Replace all http instances with https

MylesJarvis committed:
Update assessment-only-providers.md

MylesJarvis committed:
Update assessment-only-providers.md

MylesJarvis committed:
Update assessment-only-providers.md

MylesJarvis committed:
Update assessment-only-providers.md

MylesJarvis committed:
Update assessment-only-providers.md

MylesJarvis committed:
Update assessment-only-providers.md

MylesJarvis committed:
Update assessment-only-providers.md

peteryates commented:
Nice work.

I did a sweep of all our external links and these ones are problematic:

### Not found

~~https://digital.ucas.com/search: 404~~
https://www.bradfordcollege.ac.uk/qts-assessment: 404
https://www.futuretraining.org/assessment-only/: 404
https://www.tes.com/institute/assessment-only-route-qts?utm_source=nctl&utm_medium=referral&utm_content=AO-get-into-teaching-page&utm_campaign=web-referral: 404
https://www.southampton.ac.uk/education/postgraduate/itt_courses/assessment_only.page: 404
https://www.carmel.org.uk/: 000

### Bad certificate

https://aomscitt.co.uk/assessment-only/: 000
https://www.educateteachertraining.co.uk/: 000
https://www.ctt.ac.uk/: 000
https://secondary.billericayscitt.com/assessment-only-route-2/: 000
https://warwick.ac.uk/fac/soc/cte/professionaldevelopment/assessmentonly/: 000
https://www.ctt.ac.uk/: 000
https://www.glftt.org/: 000
https://www.henrymaynardtraining.co.uk/: 000
https://www.nett.org.uk/: 000
https://www.thecambridgepartnership.co.uk/: 000
https://www.snitt.co.uk/: 000
https://www.stjosephstrentvale.com/teaching-school/partnership/: 000

The 'Bad certificate' ones need changing back to `http` - it might also be worth notifying them that their sites aren't set up correctly.

The 'Not found' ones probably need replacing altogether.

peteryates committed:
Fix some assessment-only provider links that 404

Southampton Uni look like they've stopped offering an assessment-only so
that's been removed for the time being.

jebw reviewed (approved):
LGTM
